blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+
+<html lang = "en">
+
+<head>
+    <title>Blog - Adopt-a-Dog</title>

```

Comment:
you're missing the meta tag referenced in week one. You'll want to add it to all HTML pages

Reply:
Added for next PR